### PR TITLE
Send Search Component

### PR DESCRIPTION
**Pull request by youngkidwarrior**
Added a temporary Search Component for the send page.

Should probably focus on getting this replaced with a colloquial search feature that can link to an arbitrary url with the results

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/392?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#392** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/392?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#391** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/391?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#390** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/390?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#388** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/388?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#387** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/387?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#386** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/386?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#385** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/385?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#384** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/384?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#383** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/383?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#382** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/382?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#381** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/381?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#380** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/380?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#379** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/379?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/send/components/SendTopNav.tsx
Comment (0xBigBoss) [inline packages/app/features/send/components/SendTopNav.tsx:1]: image

File: packages/app/features/send/screen.tsx [hunk 1]
```diff
@@ -1,21 +1,162 @@
...
+      }}
+    >
+      <H4 theme={'alt2'}>Results</H4>
+      {results.length === 0 && <Text>No results for {query}... 😢</Text>}
+      {results.map((result) => (
+        <Link
+          key={result.tag_name}
+          href={`/send?recipient=${result.tag_name}&sendToken=${tokenParam || ''}&amount=${
+            amountParam || ''
+          }`} // @TODO: we need a more typesafe way of workong with query params in links
```
Comment (0xBigBoss) [inline packages/app/features/send/screen.tsx:107]: try 

```ts
console.log(new URLSearchParams({ ...params, recipient: result.tag_name }).toString() )
```

also maybe solito can help with this a bit more

File: packages/app/features/send/screen.tsx [hunk 2]
```diff
@@ -1,21 +1,162 @@
...
+      <SchemaForm
+        form={form}
+        defaultValues={{ query: '' }}
+        onSubmit={() => {
+          // noop
+        }}
+        schema={SearchSchema}
+        props={{
+          query: {
+            placeholder: 'Name, $Sendtag, Phone, Email',
```
Comment (0xBigBoss) [inline packages/app/features/send/screen.tsx:146]: email doesn't make sense here, sry i missed that earlier.

**Review by 0xBigBoss — APPROVED** [review]

File: packages/app/features/send/screen.tsx (hunk 2 above)
Comment (0xBigBoss) [reply packages/app/features/send/screen.tsx:146]: neither does name

File: packages/app/features/send/components/SendTopNav.tsx
Comment (youngkidwarrior) [reply packages/app/features/send/components/SendTopNav.tsx:1]: Adding to the pile so we can burn it down

File: packages/app/features/send/screen.tsx (hunk 1 above)
Comment (youngkidwarrior) [reply packages/app/features/send/screen.tsx:107]: Yeah Search params are good but still isn't really router aware. Def a place solito could upgrade itself I think

**Comment by youngkidwarrior** [comment]
### Merge activity

* **May 11, 5:36 AM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/392).

**Pull request merged**